fk/secretsend_test.go

```diff
@@ -109,6 +109,28 @@ func TestGetSecretFromFile(t *testing.T) {
 		assert.Equal(t, expectedErr, err)
 	})
 
+	t.Run("returns error if file is binary", func(t *testing.T) {
```

Comment:
FYI *all* files are binary, I think the test really means "returns error if file won't decode as UTF-8"